solutions/microservices_azure_aks/index.asciidoc

```diff
@@ -54,7 +54,7 @@ The picture below summarizes some of the services mentioned above:
 image::aks_overview.png[AKS Overview, width=794, height=568]
 
 [.internal]
-solution_microservices_azure_aks_infra_detailed_native_setup
+microservices_azure_aks_infra_detailed_native_setup
```

Comment:
Internal snippets that are used in solutions currently must be prefixed with `solution_`, because of technical reasons in a synchronization action.